backend/app/ai/utils/ai_search.py

```
@@ -48,28 +50,69 @@ def generate_keywords(query: str) -> dict:
 
     try:
         text = response.choices[0].message.content
-        print(text)
+        print("Generated response:", text)
         json_text = re.search(r"{.*}", text, re.DOTALL)
         if json_text:
             return json.loads(json_text.group())
         else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)